lm_eval/api/task.py

```diff
@@ -999,7 +1001,33 @@ def construct_requests(
             return request_list
 
         elif self.OUTPUT_TYPE == "generate_until":
-            arguments = (ctx, self.config.generation_kwargs)
+            if hasattr(self.config, 'generation_kwargs_sampling'):
+                arguments = (ctx, self.config.generation_kwargs)
+                request_list = [
+                    Instance(
+                        request_type=self.OUTPUT_TYPE,
+                        doc=doc,
+                        arguments=arguments,
+                        idx=0,
+                        **kwargs
+                        ),
+                ]
+
+                sampling_arguments = (ctx, self.config.generation_kwargs_sampling)
```

Comment:
Hi, @haileyschoelkopf, the main difference here is generation_kwargs_sampling is different to generation_kwargs. so I build multiple requests here. If you have better suggestions, please let me know. Thank you.